tests/css/atrule/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -4392,6 +4397,11 @@ B
 }
 @supports selector(:not(.b)) {
 }
+@supports not selector(: after) {
```

Comment:
Auch, indeed.

Reply:
If `func` can only checked in parseValue, why not try add a  exception, not add `value-` prefix.

Reply:
if I understood you correctly, I did that now :)

Reply:
Seems working? Maybe we  don't need that flag, how about check `type` starts with `selector-`? Thoughts?

Reply:
`selector-selector` exists, this would break that.

( assuming you mean 
```
 if (
            !node[key].startsWith(prefix) &&
            !node[key].startsWith("selector")
          ) {
```
)

Reply:
Alternative solution: passing a regex to `addTypePrefix` to test and skip adding prefix if matches.


Like so : 
```
function addTypePrefix(node, prefix, skipPrefix) {
  if (node && typeof node === "object") {
    delete node.parent;
    for (const key in node) {
      addTypePrefix(node[key], prefix, skipPrefix);
      if (key === "type" && typeof node[key] === "string") {
        if (
          !node[key].startsWith(prefix) &&
          (!skipPrefix || !skipPrefix.test(node[key]))
        ) {
          node[key] = prefix + node[key];
        }
      }
    }
  }
  return node;
}
```

called like
```
  return addTypePrefix(parsedResult, "value-", /selector/);
```

in parseValue


I don't have a strong opinion which way is better, so I'd like to know if you have a preference either way.